dronegen/mac.go

```diff
@@ -270,18 +295,45 @@ func darwinTagBuildCommands() []string {
 		`rustup override set $RUST_VERSION`,
 		`make clean release OS=$OS ARCH=$ARCH`,
 	}
+
+	if b.hasTeleportConnect() {
+		commands = append(commands,
+			`cd $WORKSPACE_DIR/go/src/github.com/gravitational/webapps`,
+			`yarn install && yarn build-term && yarn package-term`,
+		)
+	}
+
+	return commands
 }
 
-func darwinTagCopyPackageArtifactCommands() []string {
-	return []string{
+func darwinTagCopyPackageArtifactCommands(b buildType) []string {
+	commands := []string{
 		`set -u`,
 		`cd $WORKSPACE_DIR/go/src/github.com/gravitational/teleport`,
 		// copy release archives to artifact directory
 		`cp teleport*.tar.gz $WORKSPACE_DIR/go/artifacts`,
 		`cp e/teleport-ent*.tar.gz $WORKSPACE_DIR/go/artifacts`,
-		// generate checksums (for mac)
+	}
+
+	// copy Teleport Connect artifacts
+	if b.hasTeleportConnect() {
+		commands = append(commands,
+			`cd $WORKSPACE_DIR/go/src/github.com/gravitational/webapps/packages/teleterm/build/release`,
+			`cp teleterm*.dmg $WORKSPACE_DIR/go/artifacts`,
```

Comment:
The artifacts don't start with `teleterm`, they start with whatevers in `packages/teleterm/package.json` under the `productName` key. Right now it's `Teleport Connect Preview`.

In theory we could read that with something like jq, which is available in teleterm dockerfile, but I'm not sure about Drone. We'll going to change it only one more time though with the GA release, so idk if it even makes sense to do that.

Reply:
Thanks. I agree since it's not going to change often let's just hard code it.